### DISCUSSION TITLE
What terminal for VT100 menu on osx?

### DISCUSSION THREAD

**Discussion by baesek** [marcel-licence/esp32_midi_sampler#71] [Q&A]
Hi,
It took me long time to find out how to get that menu shown in marcel's videos on my mac computer (os: catalina 10.15.7). Now i use minicom. All other programs like MacWise or the screen command did not do the line break to show the menu correctly.
Is there anyone who uses a mac and can advise me what program is best for mac? The Program TeraTerm which is the proposal in the comments is not available for osx.

Thank you!
Paul

**Comment by marcel-licence** [comment]
Hi Paul,

thank you for your feedback. I could also imagine that the line break could be changed/updated. You should be able to use \n, \r and also \r\n. 
It would be good to get some more feedback on this topic.

Best regards,
Marcel
